extensions/kafka-streams/deployment/src/main/resources/dev-ui/qwc-kafka-streams-topology.js

```diff
@@ -0,0 +1,124 @@
+import { QwcHotReloadElement, html, css } from 'qwc-hot-reload-element';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { JsonRpc } from 'jsonrpc';
+
+import { Graphviz } from "https://cdn.jsdelivr.net/npm/@hpcc-js/wasm/dist/graphviz.js";
```

Comment:
Here you can use `import { Graphviz } from "@hpcc-js/wasm/graphviz";` and add this to deployment/pom.xml:

```
<dependency>
        <groupId>org.mvnpm.at.hpcc-js</groupId>
        <artifactId>wasm</artifactId>
        <version>2.14.1</version>
        <scope>runtime</scope>
      </dependency>
```

You can also add above to build parent pom dependency management and reference in the extension pom without the version